microsetta_interface/templates/new_participant.jinja2

```diff
@@ -108,6 +120,61 @@
         disableInputOnSubmit("#7-12-form", "#7-12-submit_button", "{{ _('Saving...') }}");
         disableInputOnSubmit("#13-17-form", "#13-17-submit_button", "{{ _('Saving...') }}");
         disableInputOnSubmit("#18-form", "#18-submit_button", "{{ _('Saving...') }}");
+        $("#home_page").click(function(){
+            window.open('{{home_url}}', '_self')
+        })
+        $("#create_source").click(function(){
+            //window.open('{{post_url}}', '_self')
+            //$("#source_modal").modal('hide');
+            $("#"+$(this).attr('form_id'))[0].submit();
+        })
+        $("#18-form, #0-6-form, #7-12-form, #13-17-form").submit(function(e){
+            e.preventDefault();
+            $form_id = $(this).attr('id');
+            if($form_id == "0-6-form"){
+                valid = validate06("0-6-form");
+            } else if ($form_id == "7-12-form") {
+                valid = validate712("7-12-form");
+            } else if ($form_id == "13-17-form") {
+                valid = validate1317("13-17-form")
+            } else if ($form_id == "18-form") {
+                valid = validate18("18-form")
+            }
+
+            var duplicate_source_check_msg = `It looks like you're creating a new source that may
+                  be similar or the same as an existing source.
+                  If this is the same person as an existing source,
+                  please consider providing new samples or survey
+                  responses under that source`;
```

Comment:
Can you please wrap this string in gettext() shorthand ( {{ _('It looks like...') }} ) so PyBabel can pick it up for translation?